Libraries/Components/ScrollResponder.js

```diff
@@ -112,6 +113,15 @@ type State = {
 };
 type Event = Object;
 
+function isTagInstanceOfTextInput(tag) {
+  var instance = getInstanceFromNode(tag);
+  return instance && (
+    instance.constructor.displayName === 'AndroidTextInput' ||
```

Comment:
Have you tried using `._currentElement.type`? While it's prefixed, it's safe with current reconciler implementation because this field is used as "semi public" throughout the reconciler codebase. It will break with Fiber reconciler but this doesn't matter right now because a lot of other things will break too, and we'll change them together. As long as you have a test verifying your fix works without reaching into private APIs.